pkg/apis/provisioning/v1alpha5/provisioner.go

```diff
@@ -46,6 +46,8 @@ type ProvisionerSpec struct {
 	// Termination due to expiration is disabled if this field is not set.
 	// +optional
 	TTLSecondsUntilExpired *int64 `json:"ttlSecondsUntilExpired,omitempty"`
+	// Defines a set of bounds that Karpenter obeys when provisioning capacity.
```

Comment:
Go comment style starts with the thing you've exported
```suggestion
	// Limits define a set of bounds that Karpenter obeys when provisioning capacity.
```